tests/unit/loader/test_file_system_csv_loader.py

```diff
@@ -146,6 +146,6 @@ def test_loading_with_list_of_objects(self) -> None:
             "test_column_comment",
             "test_owner",
         ]
-        expected.extend(",".join(expected_values) * 5)
+        expected.extend([",".join(expected_values)] * 5)
```

Comment:
was this breakage the result of some manual style fixes? if so, any particular areas where closer review would be helpful?

Reply:
Oh, yes that breakage was just the result of https://github.com/amundsen-io/amundsendatabuilder/pull/405/commits/372702c41bff8af9300d9477ebac2e54197e4067 . 

If you look through the commits, I made a few manual adjustments to make the code more Black-friendly, so if you want to review all manual changes, all commits except `run black` are manual. That being said, the unit tests caught that problem, and they're passing now, so I doubt there are any other issues.